pkg/app/server/service/apiservice/service.proto

```diff
@@ -39,6 +39,7 @@ service APIService {
 
     rpc EnablePiped(EnablePipedRequest) returns (EnablePipedResponse) {}
     rpc DisablePiped(DisablePipedRequest) returns (DisablePipedResponse) {}
+    rpc RestartPiped(RestartPipedRequest) returns (RestartPipedResponse) {}
```

Comment:
Adding this gRPC here means that we add a new gRPC to the cli API (API for the pipectl requests) too, while the PR title means to add gRPC only for Web API 👀  Since we discussed before that we only want to make this function (restart the piped) available via the web console, I think this one is unneeded. wdyt?

Reply:
fyi, we have 3 services under the `app/server/service` dir which are:
- apiservice: define gRPCs for pipectl (cli) requests
- webservice: define gRPCs for web requests
- pipedservice: define gRPCs for piped (agent) requests

Reply:
Oops, I mistakingly add rpc on that service. 
As you said, this PR add gRPC only for Web API so I'll delete it :bow:

Reply:
could you rerun `make gen/code`